kolibri/plugins/coach/assets/src/modules/lessonResources/handlers.js

```diff
@@ -148,17 +148,19 @@ export function showLessonResourceContentPreview(store, params) {
 
 export function showLessonSelectionContentPreview(store, params, query = {}) {
   const { classId, lessonId, contentId } = params;
+
   return store.dispatch('loading').then(() => {
     const pendingSelections = store.state.lessonSummary.workingResources || [];
+
     return Promise.all([
       _prepLessonContentPreview(store, classId, lessonId, contentId),
       store.dispatch('lessonSummary/updateCurrentLesson', lessonId),
     ])
       .then(([contentNode, lesson]) => {
         // TODO state mapper
-        const preselectedResources = lesson.resources.map(({ contentnode_id }) => contentnode_id);
-
+        const preselectedResources = lesson.resources;
```

Comment:
So this was the main cause, that this should have been an array of Content Node objects instead of an array of strings? We had done a refactor that replaced our representation of Lesson Contents from the IDs of the nodes to the full objects, so it must not have made it here.